gdc_client/upload/client.py

```diff
@@ -180,26 +180,24 @@ def __init__(
         self.upload_id = None
         self.debug = debug
         self.processes = processes
-        self.upload_part_size = int(
-            max(upload_part_size, MIN_PARTSIZE) / PAGESIZE + 1
-        ) * PAGESIZE
+        self.upload_part_size = (
+            int(max(upload_part_size, MIN_PARTSIZE) / PAGESIZE + 1) * PAGESIZE
+        )
```

Comment:
did you make this change because it's possible for `PAGESIZE` to not be an int?

Reply:
i think this is just a black formatting change